parser/Parser.cc

```diff
@@ -31,24 +31,18 @@ class ErrorToError {
         u4 maxOff = file.data(gs).source().size();
         file.data(gs).hasParseErrors = true;
         for (auto &diag : diagnostics) {
-            string_view level = "unknown"sv;
             switch (diag.level()) {
                 case ruby_parser::dlevel::NOTE:
                 case ruby_parser::dlevel::WARNING:
                     continue;
                 case ruby_parser::dlevel::ERROR:
-                    level = "Error"sv;
-                    break;
                 case ruby_parser::dlevel::FATAL:
-                    level = "Fatal"sv;
                     break;
-                default:
-                    Exception::notImplemented();
             }
             core::Loc loc(file, translatePos(diag.location().beginPos, maxOff - 1),
                           translatePos(diag.location().endPos, maxOff));
             if (auto e = gs.beginError(loc, core::errors::Parser::ParserError)) {
-                e.setHeader("Parse {}: {}", level, fmt::format(dclassStrings[(int)diag.error_class()], diag.data()));
+                e.setHeader("{}", fmt::format(dclassStrings[(int)diag.error_class()], diag.data()));
```

Comment:
I think I might be missing context from the original PR, but why remove the `Parse Error` prefix?

Reply:
That's the change requested in https://github.com/sorbet/sorbet/pull/1670#discussion_r322410454.

Honestly I like it because my target goal is to rewrite sorbet/parser to match whitequark/parser, and so this way there's no need to do any preprocessing of error messages before backporting tests from whitequark/parser

Reply:
👍 Thanks!